go.mod

```diff
@@ -1,7 +1,7 @@
-go 1.21
+go 1.22
```

Comment:
For consistency should we set this to 1.22.0 so that the toolchain line is removed like in the other projects?

I was thinking about updating the warning comment here to always say we never want the toolchain in go.mod

Reply:
Both done.

Thanks!